RAMP/unpacker.py

```diff
@@ -27,14 +26,18 @@ def __str__(self):
 
 
 def unpack(bundle, max_bundle_size_mb=None):
-    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes]]
+    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes], Dict[str, Any]]
     """
     Unpacks a bundled module, performs sanity validation on bundle.
     both the module metadata and the actual module are returned
     :rtype: tuple
     """
+    deps_files = dict()
     try:
         with ZipFile(bundle) as zf:
+            for filename in zf.namelist():
+                if filename.startswith("deps"):
+                    deps_files[filename] = zf.open(filename)
```

Comment:
1. the docstring can be updated
2. the type annotation can be more specific (IO)
3. maybe document an example of the expected zip structure and the matching output?